frontend/sass/norent/styles.scss

```diff
@@ -61,4 +84,134 @@ $jf-navbar-content: $black;
       color: $link;
     }
   }
+
+  // Default spacing between paragraphs
+  p:not(:last-child) {
+    margin-bottom: 0.5rem;
+  }
+
+  // Set all Bulma container classes to have max width:
+  .container {
+    max-width: 800px;
+    &.jf-tight-container {
+      max-width: 600px;
+    }
+    &.jf-wide-container {
+      max-width: 1000px;
+    }
+  }
+
+  // Make all media icons have vertical centering
+  // as well as smaller margins on mobile
+  .media .media-left,
+  .media .media-right {
+    margin-top: auto;
+    margin-bottom: auto;
+    @media screen and (max-width: $tablet) {
+      figure {
+        margin-left: 1rem;
+        margin-right: 1rem;
+      }
+    }
+  }
+}
+
+// CUSTOM CLASSES
+
```

Comment:
I bet some of these custom classes could be useful throughout the TP. Any thoughts?

Reply:
Yeah, for the ones that are super generic I think it's fine to put them in some kind of "utility classes" or "helpers" file.  There are a few of these in the JustFix site's `styles.scss` as well, e.g. `jf-has-text-underline` and `jf-word-glue`.  Feel free to do that in this PR or file an issue to remind us to do it later!

Reply:
Done! https://github.com/JustFixNYC/tenants2/issues/1197